tools/run_session

```diff
@@ -35,8 +34,7 @@ if [[ ! $(pgrep -l mosquitto | head -n1 | cut -d' ' -f2) == mosquitto ]]; then
     "$MOSQUITTO" &> ${TOMCAT_TMP_DIR}/mosquitto.log &
 fi
 
-# On macOS, uuidgen produces uppercase UUIDs, so we pipe the output through tr
-# to get uniform behaviour on macOS and Linux.
+# UUIDs: uniform behaviour on macOS and Linux.
```

Comment:
I think it would be good to leave the more detailed comment here for future developers.

Reply:
Fixed. I wondered if my original explanation in the commentary had been too verbose so I shortened it. I restored it back to that original explanation.